lib/manageiq-api.rb

```diff
@@ -1,2 +1,25 @@
-require "manageiq/api/engine"
-require "manageiq/api/version"
+
+require "zeitwerk"
+loader = Zeitwerk::Loader.for_gem(warn_on_extra_files: false)
+loader.ignore(__FILE__)
+
+# Add non-standard root directory to the path
+loader.push_dir("#{__dir__}/../app/controllers")
+
+# This tells the loader to not expect Services namespace
+loader.collapse("#{__dir__}/services")
+
+loader.ignore("#{__dir__}/api/api_config.rb")  # Dynamically loads all Vmdb::Plugins - assumes we're mounted in manageiq
+
+# These inflectors teach the loader our naming convention here
+loader.inflector.inflect(
+  "manageiq"  => "ManageIQ",
```

Comment:
Similarly, this is a pain to do everywhere - wonder if there's a way to inherit this from core (perhaps when core is finally converted we can come back and remove the redundant stuff)

Reply:
yeah, that's why I wanted to try out a few repos to see what's common or different and to see if there's a better way to do it.